Cargo.toml

```diff
@@ -62,6 +62,7 @@ rustversion = "1.0.7"
 serde_json = "1.0.68"
 serde_plain = "1.0.0"
 surf = { version = "2.3.1", optional = true }
+uuid = { version = "1.12.1", features = ["v4"] }
```

Comment:
Each SDK should provide a platform specific UUID generation. From my research uuid has been moved out of stdlib into a crate.